api/scpca_portal/models/project.py

```diff
@@ -269,6 +269,13 @@ def get_download_config_file_paths(self, download_config: Dict) -> List[Path]:
             file_path for file_path in data_file_path_objects if file_path.parent.name != "merged"
         ]
 
+    def get_valid_download_configs(self) -> List[str]:
```

Comment:
Let's break this up into two different properties `project.valid_download_configs` and `project.valid_download_config_names`